Fixes errors like below with eglibc 2.16

libudev/libudev-util-private.c:348:16: error: 'PRIO_PROCESS' undeclared (first use in this function)

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending

Index: udev-164/libudev/libudev-util-private.c
===================================================================
--- udev-164.orig/libudev/libudev-util-private.c	2010-08-03 02:42:26.000000000 -0700
+++ udev-164/libudev/libudev-util-private.c	2012-07-03 10:53:35.082346684 -0700
@@ -21,6 +21,7 @@
 #include <grp.h>
 #include <sys/wait.h>
 #include <sys/param.h>
+#include <sys/resource.h>
 
 #include "libudev.h"
 #include "libudev-private.h"
Index: udev-164/udev/udevd.c
===================================================================
--- udev-164.orig/udev/udevd.c	2010-10-13 04:08:29.000000000 -0700
+++ udev-164/udev/udevd.c	2012-07-03 10:53:19.214345914 -0700
@@ -43,6 +43,7 @@
 #include <sys/ioctl.h>
 #include <sys/inotify.h>
 #include <sys/utsname.h>
+#include <sys/resource.h>
 
 #include "udev.h"
 #include "sd-daemon.h"
